microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -19,6 +19,18 @@ On Windows 11, widgets appear in the widgets board which you open from the left
 In Windows 11, Progressive Web Apps (PWAs) can define widgets, update them, and handle user interactions within them.
 
 
+<!-- ------------------------------ -->
+#### Requires developing a PWA as a widget
+
+For a PWA-driven widget, you build the widget experience by using the web app manifest, Adaptive Card templates, and JavaScript, rather than putting an existing PWA into a widget.  A PWA-driven widget is a separate experience, distinct from PWAs in the sidebar, which takes the entire PWA and puts it in the sidebar.
+
+To build a PWA-driven widget and deliver it via the Microsoft store, no C++/C# code is required.  Once you've produced the PWA-driven widget, and can successfully install and run the PWA-driven widget from a public endpoint, you can package the app by using [PWABuilder.com](https://www.pwabuilder.com) and ship the app to the Microsoft store without requiring any additional code.  A PWA-driven widget must be installed and run from a public endpoint, because the PWABuilder.com tool doesn't work with localhost.
```

Comment:
1. There are 3 instances of "PWA-driven widget" in here in addition to the 2 instances in the previous paragraph. WDYT about just calling it a "widget" after the first use?

2. Since you don't have to install the widget to run PWABuilder--it just needs to be _installable_--maybe the last sentence should be:
> The PWA backing the widget must be installable from a public endpoint because PWABuilder.com does not support packing apps from localhost.

Also note the technicality of the PWA being installable and packaged as opposed to the widget, since the widget isn't a standalone entity.

Reply:
1. Spelled out "PWA-driven widget" only in 1st occurrence in a paragraph.
2. Changed last sentence.
3. Entered hidden comment "The PWA (not the widget) is what is installable and packaged, because it's a standalone entity (unlike a widget)."